pkgs/desktops/gnome-3/core/mutter/default.nix

```diff
@@ -16,6 +16,13 @@ stdenv.mkDerivation rec {
     updateScript = gnome3.updateScript { packageName = "mutter"; attrPath = "gnome3.mutter"; };
   };
 
+  patches = [
+    (fetchpatch {
```

Comment:
Perhaps a comment with why we have this patch?

Reply:
Shall I just duplicate the commit message here in a comment?

Reply:
Usually just a link to the issue should be fine.

You may also mention briefly
> Fixes direction of the desktop switching animation when using workspace grid extension with desktops arranged horizontally.

Reply:
Thanks! Quoted the issue link and issue description.